spec/index.bs

```diff
@@ -2976,17 +2958,67 @@ type and each [=binding type=] has an associated [=internal usage=], given by th
         <td>[=internal usage/storage-write=]
 </table>
 
+<div algorithm="binding cost">
+    The <dfn>binding cost</dfn> for a [=list=] |entries| of {{GPUBindGroupLayoutEntry}} values
```

Comment:
I don't have the same mental association of "cost" meaning a potentially fluctuating thing that @kvark mentioned, but for whatever reason the term "cost" here still reads awkwardly to me when paired with "limits" of a particular resource. I don't have a good explanation as to why.

Other potential terms that I can think of:
 - binding requirements
 - binding uses 
 - binding counts